pages/index.html

```diff
@@ -36,46 +37,61 @@
 
     <!-- Body -->
     <body>
-        <div class="container">
-            <div class="row">
-              <div class="col-md-6">
-                <img src="BizEverywhereSign.jpg" class="mx-auto d-block">
-              </div>
-            </div>
-        </div>
-          
-        <!-- Drop down menu -->
-        <!-- <div class="dropdown">
-            <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" 
-                aria-haspopup="true" aria-expanded="false"> Menu
-            </button>
-            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-              <a class="dropdown-item" href="https://www.instagram.com/tonybanhh/">My Instagram</a>
-              <a class="dropdown-item" href="https://www.linkedin.com/in/tony-banh-46631080/">My Linkedin</a>
-              <a class="dropdown-item" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstley">Something else here</a>
-            </div>
-        </div> -->
-        <!-- Search Bar -->
-        <div class="container">
-            <!-- <form>
-                <div class="form-floating">
-                    <input type="search" id="search" class="form-control" placeholder="Searching...">
-                    <label for="search">Search</label>
-                </div>
-                <button>Submit</button>
-            </form> -->
-            <div class="card text-bg-dark">
-                <div class="card-body">
-                    This is the body
+        <div style="display: flex; flex-direction: column; min-height: 100vh; position: relative;">
```

Comment:
Inline style is not allowed. **Bootstrap has these styles**, eg: display:flex -> d-flex, min-height: 100vh -> vh-100